mlir/lib/Gradient/IR/GradientOps.cpp

```diff
@@ -28,6 +28,25 @@
 using namespace mlir;
 using namespace catalyst::gradient;
 
+//===----------------------------------------------------------------------===//
+// SymbolUserOpInterface
+//===----------------------------------------------------------------------===//
+
+void GradOp::setCalleeFromCallable(CallInterfaceCallable callee)
+{
+    (*this)->setAttr("callee", callee.get<SymbolRefAttr>());
+};
+
+void JVPOp::setCalleeFromCallable(CallInterfaceCallable callee)
+{
+    (*this)->setAttr("callee", callee.get<SymbolRefAttr>());
+};
+
+void VJPOp::setCalleeFromCallable(CallInterfaceCallable callee)
+{
+    (*this)->setAttr("callee", callee.get<SymbolRefAttr>());
+};
```

Comment:
Small but these should probably be moved to each respective heading below.

Reply:
https://github.com/PennyLaneAI/catalyst/pull/143/commits/3f9283ed6c9bac1fed84e1dd901e980bae278ef0